themes/default/views/editor/objects/summary_html.php

```diff
@@ -38,6 +38,12 @@
 <?php
 	if ($vs_display_select_html = $t_display->getBundleDisplaysAsHTMLSelect('display_id', array('onchange' => 'jQuery("#caSummaryDisplaySelectorForm").submit();',  'class' => 'searchFormSelector'), array('table' => $t_item->tableNum(), 'value' => $t_display->getPrimaryKey(), 'access' => __CA_BUNDLE_DISPLAY_READ_ACCESS__, 'user_id' => $this->request->getUserID(), 'restrictToTypes' => array($t_item->getTypeID())))) {
 ?>
+		<div id="toggleCollapsedButton">
```

Comment:
This will obviously only apply on object displays. Wondering if we should add this for other record types for consistency? Will merge anyway, we can always create another task to do that.